### PR TITLE
Add InkToolbar page

### LINKED ISSUE #297: Add InkToolbar Sample
**Is this a totally new sample, an addition to an existing sample page, or a modification of an existing sample page?**
- [x] New sample
- [ ] Addition to existing sample
- [ ] Modification to existing sample

**If this is an addition/modification to an existing sample page, which one?**

**Describe the sample**

InkToolbar is a control that contains a customizable and extensible collection of buttons that activate ink-related features relative to an InkCanvas. We have an InkCanvas sample, but no associated InkToolbar sample

**Is your sample request related to a problem? Please describe it**

Referenced by #296 

**Additional context**

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

Adds the missing InkToolbar page.
## Motivation and Context

Fixes #297 
## How Has This Been Tested?

Tested manually by starting application.
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/InkToolbarPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+using Windows.UI.Input.Inking;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/InkToolbarPage.xaml.cs]: Please remove this boilerplate comment (and the one below)

**Comment by stmoy** [comment]
image

File: XamlControlsGallery/ControlPages/InkToolbarPage.xaml.cs (hunk 1 above)
Comment (chingucoding) [reply XamlControlsGallery/ControlPages/InkToolbarPage.xaml.cs]: Totally missed that, removed now (and also unused using statements)

**Comment by stmoy** [comment]
Much thank @chingucoding !

**Comment by chingucoding** [comment]
Thank you for the wonderful meme @stmoy 😁

**Pull request merged**